style/base.css

```diff
@@ -0,0 +1,489 @@
+@font-face {
+    font-family: 'Lato Regular';
+    src: url('../assets/fonts/Lato-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+body {
+    background-color: #202542;
+    font-family: 'Lato Regular', sans-serif;
+    margin: 0;
+    line-height: 32px;
+}
+
+header a,
+footer a {
+    color: #fff;
+    text-decoration: none;
+}
+
+header {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 2rem 4rem;
+}
+
+header>address>a {
+    display: flex;
+    align-items: center;
+    font-size: 1.5rem;
+    font-style: normal;
+}
+
+header>address>a>img {
+    width: 34px;
+    margin-right: 1rem;
```

Comment:
What are your thoughts on using rem for font-size, but px for everything else? Just thinking about future contributors, I think it would be easier to compare widths, padding, margins etc if they are all in pixels.